users/templates/register.html

```diff
@@ -0,0 +1,14 @@
+ {% extends "base.html" %}
+
+{% load crispy_forms_tags %}
+{% block content %}
+<h2>Register Page</h2>
+<form method="POST" class="form-group">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <p>Already have an account? Login <a href="/login">here</a></p>
```

Comment:
I would used named URLs. E.g. `href={% url 'users:login' %}` or just `{% url 'login' %}` since you've included the auth links directly in project.